src/commands/traces/drivers/byovd.rs

```diff
@@ -2,76 +2,147 @@
 //
 // SPDX-License-Identifier: GPL-3.0-or-later
 
-use crate::{commands::Runnable, windows::users::is_administrator};
+use crate::{
+    commands::{traces::Trace, Runnable},
+    displayer::Displayer,
+    windows::users::is_administrator,
+};
 use clap::Parser;
 use serde::Deserialize;
-use std::{error::Error, path::PathBuf};
+use std::{path::PathBuf, process::ExitCode};
 use windows::{
     core::{Owned, HSTRING, PCWSTR},
-    Win32::{
-        Foundation::GENERIC_READ,
-        System::Services::{
-            CreateServiceW, OpenSCManagerW, OpenServiceW, StartServiceW, SC_HANDLE,
-            SC_MANAGER_ALL_ACCESS, SC_MANAGER_CREATE_SERVICE, SERVICE_AUTO_START,
-            SERVICE_ERROR_IGNORE, SERVICE_KERNEL_DRIVER,
-        },
+    Win32::System::Services::{
+        CreateServiceW, OpenSCManagerW, StartServiceW, SC_HANDLE, SC_MANAGER_ALL_ACCESS,
+        SC_MANAGER_CREATE_SERVICE, SERVICE_AUTO_START, SERVICE_ERROR_IGNORE, SERVICE_KERNEL_DRIVER,
     },
 };
 
 #[derive(Parser, Deserialize)]
 pub struct Byovd {
-    #[clap(required = true, help = "Name of the service")]
-    service_name: String,
+    #[clap(required = true, help = "Path to the driver")]
+    driver: PathBuf,
```

Comment:
```suggestion
    driver_path: PathBuf,
```
Just driver is no very clear

Reply:
It's not a good thing to include type in the variable name, so no for me It's pretty clear.
In computer science, everything is opinionated, but like I said, this is pretty much considered a bad practice